### ISSUE TITLE
Re-word "System is running the latest" update page text

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2115]
Thanks to forum member ednt in the following forum thread for highlighting this issue. It can be confusing for new users, who have as yet not read our docs "Update

Given recent updates / improvements to our  "System -> Software Update" page, which now links back to the docs "Update Channels" section to appreciate that the following message:

> System is running the latest Rockstor version:

pertains to the selected channel. As from 3.9.2-52 this page has been much improved with more concise text, and contextual links back to our docs and the forum; I propose that we edit this now legacy wording by removing the words "the latest" as this is overly contextual and potentially misleading. In any given channel it may be the latest, and this message is shown when no updates are available but does not account for lapsed subscriptions, or a failure to check the repos.

So in short I propose we change the above message to:

> System is running Rockstor version:

for both our currency newer Stable channel and our soon to be newer  (post 'Built on openSUSE' transition) testing channel "Software Update" page displays.

**Comment by phillxnet** [comment]
@FroggyFlox , do you agree? I was hoping our newest contributor @jonpwilson would be game to tackle this one :)
And keeping it simple helps to finally get it done as this has been a potentially misleading text element for some time now. We could link to our GitHub releases page but I wanted to keep this issue simple and that could go in another pr if need be.

**Comment by FroggyFlox** [comment]
That's a good idea, @phillxnet... 

It is true that the current wording can be misleading so the simpler the better as it usually avoids confusion.
